CalculatorTest.py

```diff
@@ -20,6 +20,9 @@ def test_run_operation_divide(self):
         with self.assertRaises(ZeroDivisionError):
             Calculator.run_operation(9, 0, '/')
 
+    def test_run_operation_modulo(self):
```

Comment:
You should also do raiseZeroDivision for mod because we can't divide by 0 and so n mod 0 should raise exception

Reply:
Agreed, raise an exception for when the second number given is 0 and the operation is division.

Reply:
I'm with you